src/inmanta/execute/scheduler.py

```diff
@@ -280,13 +280,21 @@ def anchormap(
             if anchor is not None
         )
 
-        rangetorange = [(anchor.get_location(), anchor.resolve()) for anchor in anchors]
-        rangetorange = [(f, t) for f, t in rangetorange if t is not None]
+        range_to_anchor_target = [(anchor.get_location(), anchor.resolve()) for anchor in anchors]
+        range_to_anchor_target = [(f, t) for f, t in range_to_anchor_target if t is not None]
 
         now = time.time()
         LOGGER.debug("Anchormap took %f seconds", now - prev)
 
-        return rangetorange
+        return range_to_anchor_target
+
+    def anchormap(
```

Comment:
We need a docstring here to indicate that this method exists for backwards compatibility with inmanta-vscode.